### PR TITLE
Working with time ld import command

### PR DESCRIPTION
**Pull request by ishifoev**
#138

### CODE REVIEW

File: src/api/timeld.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+require_once(__DIR__ . '/callGetEndpoint.php');
+require_once(__DIR__ . '/callEndpoint.php');
+
+
+function importTimld($data = array()) {
+    $url = $_SERVER["TIMELD_HOST"] . '/import';
+
+
+    $headers = array(
+        "Accept: Content-Type: application/x-ndjson",
```
Comment (michielbdejong) [inline src/api/timeld.php:13]: Should this be `Accept: application/x-ndjson`?

File: src/api/timeld.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once(__DIR__ . '/../../vendor/autoload.php');
+require_once(__DIR__ . '/callGetEndpoint.php');
+require_once(__DIR__ . '/callEndpoint.php');
+
+
+function importTimld($data = array()) {
```
Comment (michielbdejong) [inline src/api/timeld.php:8]: Should this be `importTimeld`?

**Review by michielbdejong — APPROVED** [review]
LGTM! I think what you built is export to Timeld, not import from Timeld, could that be?

File: src/api/timeld.php (hunk 1 above)
Comment (michielbdejong) [reply src/api/timeld.php:13]: Or if this is export *to* Timeld, maybe it should be `Content-Type: application/x-ndjson`?

**Comment by michielbdejong** [comment]
Or rather, I think this command doesn't touch the PJ database at all?

**Comment by ishifoev** [comment]
@michielbdejong It is a importing to timeld, it is not exporting.

**Pull request merged**